runtime/pkg/duration/duration.go

```diff
@@ -5,12 +5,20 @@ import (
 	"fmt"
 	"regexp"
 	"strconv"
+	"strings"
 	"time"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
```

Comment:
We should avoid importing API types into generic `pkg` packages when possible. In this case, would consider seeing if you can find a relevant stdlib type or otherwise define an enum in this package (and have a simple `switch`-convert function on the `queries` package side).